src/web/components/lib/dateTime.ts

```diff
@@ -133,28 +133,28 @@ export const makeRelativeDate = (
     if (delta < 0) {
         // Publication dates in the future are not supported
         return false;
-    } else if (opts.notAfter && delta > opts.notAfter) {
+    } if (opts.notAfter && delta > opts.notAfter) {
         // If article was published before the cutoff (notAfter) bail out
         return false;
-    } else if (delta < 55) {
+    } if (delta < 55) {
```

Comment:
weird flex but ok

Reply:
I added the comment

```
    // tslint:disable no-else-after-return because I think
    // the intention reads better with the else statements here
```
at the top of this if/else block to prevent this. It kind of looks wrong without else statements. I realise there's a return in each case so it's redundant but...

Myeh 🤷‍♂ I guess it's readable either way but we should lose the `tslint:disable no-else-after-return` if we're not using it

Reply:
No worries, I reset it and converted it to an `eslint-disable`